crypto/utils/batch_key.go

```diff
@@ -8,8 +8,7 @@ import (
 )
 
 const (
-	// TODO: move to a common constants file which
-	// can be used by utils and cmd
```

Comment:
Can you also change `/cmd/egnkey/generate.go` to use these constants from utils instead of redefining it themselves. I think this is what this TODO was meant to say. I tried just now and it seems there's no circular dependency so not sure why we redefined. Change should be straightforward.